.github/workflows/dart_ci.yml

```diff
@@ -47,43 +46,4 @@ jobs:
           dart run build_runner test -r -- -P concurrent-tests
           dart run build_runner test -r -- -P non-concurrent-tests
         timeout-minutes: 10
-        if: ${{ always() && steps.install.outcome == 'success' }}
-
-  build_legacy:
-    runs-on: ubuntu-latest
```

Comment:
It looks like this removes the run using pubspec.yaml, and the only one left uses pubspec.next.yaml. I'm unfamiliar with what purpose exactly pubspec.next.yaml serves, but if we're publishing using pubspec.yaml, we should be running CI on it.